RoosterPlanner.Api/Models/ProjectDetailsViewModel.cs

```diff
@@ -16,12 +17,12 @@ public class ProjectDetailsViewModel
         public string PictureUri { get; set; }
         public string WebsiteUrl { get; set; }
         public bool Closed { get; set; }
-        public List<RoosterPlanner.Models.Task> Tasks { get; set; }
+        public List<Task> Tasks { get; set; }
```

Comment:
Even bepalen of 'Task' wel een handige naam is. Aangezien .net ook een keyword Task heeft.

Reply:
@DRTomberge 'Assignment'?

Reply:
Laten we dit even overleggen